Xplat/src/main/java/vazkii/patchouli/mixin/AccessorComponentSerializer.java

```diff
@@ -0,0 +1,17 @@
+package vazkii.patchouli.mixin;
+
+import com.google.gson.JsonElement;
+
+import net.minecraft.core.HolderLookup;
+import net.minecraft.network.chat.Component;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+@Mixin(Component.Serializer.class)
+public interface AccessorComponentSerializer {
+	@Invoker("serialize")
+	public static JsonElement invokeSerialize(Component component, HolderLookup.Provider provider) {
+		throw new AssertionError();
+	}
+}
```

Comment:
Why is this needed? can't you use the codec with a RegistryOps<JsonElement>?

Reply:
I could, but that would basically be an exact copy of an already existing method

Reply:
still preferred to adding mixins

Reply:
I've removed the mixin in favor of copying the method